subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h

```diff
@@ -4,9 +4,12 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#ifndef RADIO_NRF5_PPI_H__
+#define RADIO_NRF5_PPI_H__
 
 #if defined(CONFIG_SOC_SERIES_NRF51X) || defined(CONFIG_SOC_COMPATIBLE_NRF52X)
 
+#include "radio_nrf5_ppi_resources.h"
```

Comment:
Move this after standard include to line after 14, leave a blank line in between.